src/IIIFPresentation/API.Tests/Integration/ModifyManifestUpdateTests.cs

```diff
@@ -0,0 +1,361 @@
+        SetCorrectEtag(requestMessage, dbManifest);
+
+        // Act
+        var response = await httpClient.AsCustomer(Customer).SendAsync(requestMessage);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.Conflict);
+    }
+    
+    [Fact]
+    public async Task PutFlatId_Update_BadRequest_IfParentAndSlugAlreadyExist_ForCollection()
+    {
+        // Arrange
+        var dbCollection = (await dbContext.Collections.AddTestCollection()).Entity;
+        var dbManifest = (await dbContext.Manifests.AddTestManifest()).Entity;
+        await dbContext.SaveChangesAsync();
+        var manifest = dbManifest.ToPresentationManifest(slug: dbCollection.Hierarchy.Single().Slug);
+        
+        var requestMessage =
+            HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Put, $"{Customer}/manifests/{dbManifest.Id}",
+                manifest.AsJson());
+        SetCorrectEtag(requestMessage, dbManifest);
+
+        // Act
+        var response = await httpClient.AsCustomer(Customer).SendAsync(requestMessage);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.Conflict);
+    }
+    
+    [Fact]
+    public async Task PutFlatId_Update_BadRequest_IfParentAndSlugAlreadyExist_ForManifest()
+    {
+        // Arrange
+        var duplicateId = $"id_{PutFlatId_Update_BadRequest_IfParentAndSlugAlreadyExist_ForManifest}";
+        var duplicateManifest = (await dbContext.Manifests.AddTestManifest(duplicateId)).Entity;
+        var dbManifest = (await dbContext.Manifests.AddTestManifest()).Entity;
+        await dbContext.SaveChangesAsync();
+        var manifest = dbManifest.ToPresentationManifest(slug: duplicateManifest.Hierarchy.Single().Slug);
+        
+        var requestMessage =
+            HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Put, $"{Customer}/manifests/{dbManifest.Id}",
+                manifest.AsJson());
+        SetCorrectEtag(requestMessage, dbManifest);
+
+        // Act
+        var response = await httpClient.AsCustomer(Customer).SendAsync(requestMessage);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.Conflict);
+    }
+
+    [Fact]
+    public async Task PutFlatId_Update_BadRequest_WhenParentIsInvalidHierarchicalUri()
+    {
+        // Arrange
+        var dbManifest = (await dbContext.Manifests.AddTestManifest()).Entity;
+        await dbContext.SaveChangesAsync();
+        var manifest = dbManifest.ToPresentationManifest(parent: "http://different.host/root");
+        
+        var requestMessage =
+            HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Put, $"{Customer}/manifests/{dbManifest.Id}",
+                manifest.AsJson());
+        SetCorrectEtag(requestMessage, dbManifest);
+
+        // Act
+        var response = await httpClient.AsCustomer(Customer).SendAsync(requestMessage);
+
+        // Assert
+        var error = await response.ReadAsPresentationResponseAsync<Error>();
+
+        // Assert
```

Comment:
There's two `assert` comments here

Reply:
This is first part of fix for #62 so will remove in follow up change